app/grandchallenge/components/validate_scripts/validate_biom.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from pathlib import Path
+
+
+class ValidationError(Exception):
```

Comment:
>It is probably worth changing the name of the ValidationError in case a future developer tries to import it and picks the wrong one.

Oh yeah! I fell into that trap a few times of blindly accepting an IDE-suggested import.